packages/fiori/src/ShellBar.hbs

```diff
@@ -89,6 +90,7 @@
 						<ui5-button
 							@click={{_handleCancelButtonPress}}
 							class="ui5-shellbar-button"
+							data-ui5-stable="cancelSearch"
```

Comment:
**[suggestion (non-blocking)](https://conventionalcomments.org/#labels):** That's good that this `"cancelSearch"` value for `data-ui5-stable` doesn't reference the role of the element, but should we be using kebab case?

```suggestion
							data-ui5-stable="cancel-search"
```